design/Accounts_signup.fxml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.PasswordField?>
+<?import javafx.scene.control.TextField?>
+<?import javafx.scene.image.Image?>
+<?import javafx.scene.image.ImageView?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.Region?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.text.Text?>
+
+<HBox
+    prefHeight="400"
+    xmlns:fx="http://javafx.com/fxml">
+    <VBox
+        prefWidth="250"
+        styleClass="left-container">
+        <Region prefHeight="50" />
+    </VBox>
+    <VBox
+        prefWidth="550"
+        styleClass="content">
+        <Text styleClass="heading1"> Sign Up </Text>
+        <HBox>
+            <Label text="Email" />
+            <Region minWidth="70" />
+            <Text
+                fx:id="emailFeedback"
+                styleClass="feedback">
+            </Text>
+        </HBox>
+        <TextField
+            fx:id="email"
+            promptText="Enter email here"
+            onAction="#handleSignup" />
+        <HBox>
+            <Label text="Username" />
+            <Region minWidth="40" />
+            <Text
+                fx:id="usernameFeedback"
+                styleClass="feedback">
+            </Text>
+        </HBox>
+        <TextField
+            fx:id="username"
+            promptText="Enter username here"
+            onAction="#handleSignup" />
+        <HBox>
+            <Label text="Password" />
+            <Region minWidth="40" />
+            <Text
+                fx:id="passwordFeedback"
+                styleClass="feedback">
+            </Text>
+        </HBox>
+        <PasswordField
+            fx:id="password"
+            promptText="Enter password here"
+            onAction="#handleSignup" />
+        <Text
+            wrappingWidth="500"
+            textAlignment="right"
+            styleClass="link-text"
+            onMouseClicked="#setLoginPage"
+            onKeyPressed="#setLoginPageKeyboard"> Already have an account? Login instead!
```

Comment:
Log in, not login. Log in = verb, login = noun.

http://grammarist.com/spelling/log-in-login/